static/src/js/containers/SearchSidebarHeaderContainer/SearchSidebarHeaderContainer.js

```diff
@@ -8,22 +8,42 @@ import SearchSidebarHeader from '../../components/SearchSidebar/SearchSidebarHea
 import actions from '../../actions/index'
 import { locationPropType } from '../../util/propTypes/location'
 
+export const mapStateToProps = (state) => ({
+  portal: state.portal,
+  projectCollectionIds: state.project.collections.allIds
+})
+
 export const mapDispatchToProps = (dispatch) => ({
   onFocusedCollectionChange:
-    (collectionId) => dispatch(actions.changeFocusedCollection(collectionId))
+    (collectionId) => dispatch(actions.changeFocusedCollection(collectionId)),
+  onChangePath: (path) => dispatch(actions.changePath(path)),
+  onLoadPortalConfig:
+  (portalId) => dispatch(actions.loadPortalConfig(portalId))
 })
 
-export const SearchSidebarHeaderContainer = ({ location, onFocusedCollectionChange }) => (
+export const SearchSidebarHeaderContainer = ({
+  location,
+  onFocusedCollectionChange,
+  portal,
+  onChangePath,
+  onLoadPortalConfig
+}) => (
   <SearchSidebarHeader
     location={location}
     onFocusedCollectionChange={onFocusedCollectionChange}
```

Comment:
This onFocusedColletionChange should be removed the child component is not using it